row_012_DB/row_012_DB_unsafe.py

```
@@ -0,0 +1,417 @@
+    cursor.execute(f"SELECT * FROM users_stations;")
+    result = cursor.fetchall()
+    connection.close()
+    return result
+
+
+def drop_user(user_id: int) -> None:
+    connection = sql.connect('userbase.db')
+    cursor = connection.cursor()
+
+    cursor.execute(f"delete from users"
+                   f" where user_id = '{user_id}';")
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)